### ISSUE TITLE
Add more hosts

### ISSUE THREAD

**Issue by snhv** [AdAway/AdAway#799]
Hi I'd like to contribute a few more hosts:

* used by WPS Office:
counter.kingsoft.com
analytics.mopub.com
img.particlenews.com
openapi.particlenews.com
stats.appsflyer.com
t.appsflyer.com

* used by Cheetah apps:
ms.cmcm.com

* used by Opera apps (proof that Opera now belongs to Chinese internet police patrol):
cp.oupeng.com
startup.oupeng.com

**Comment by jawz101** [comment]
a.fiksu.com
api.segment.io
cdn.segment.com
dc.services.visualstudio.com
e.crashlytics.com
mobile-service.segment.com
settings.crashlytics.com
analytics-prod-ingress.bricknet.com
tags.otherlevels.com
admin.brightcove.com
app.adjust.com
fonts.googleapis.com
brahe.apptimize.com
md-a-c.apptimize.com
appload.ingest.crittercism.com
s.amazon-adsystem.com
device-metrics-us-2.amazon.com
14716.engine.mobileapptracking.com
stats.appsflyer.com
t.appsflyer.com
t.singular.net
control.kochava.com
www.googleadservices.com
www.googletagmanager.com
secure.insightexpressai.com
*.36a9.ip4.static.sl-reverse.com/ (... several resolve for mixpanel analytics svcs through sl-reverse.com and mixpanel has something like a block of 128 IP's through sl-reverse.com with random names like the above)

**Comment by DJCrashdummy** [comment]
incidentally i noticed some additional domains related to _WPS Office_, which were contacted while not even using it (for far more than a day) - and more than a dozen reboots in between:
`cloudservice14.kingsoft-office-service.com`
<s>`counter.kingsoft.com`</s> already mentioned by snhv
`movip.wps.com`

**Comment by patrickdrd** [comment]
thanks!

@jawz101 you're aware that you cannot use wildcards in adaway though I guess

**Comment by jawz101** [comment]
yeah.  I didn't know how to express that there's a bunch out there.  I wanted to play around with this project to enumerate their subdomains https://github.com/jonluca/Anubis
